README.md

```diff
@@ -3,6 +3,9 @@ funKTionale
 
 Functional constructs and patterns for [Kotlin](http://kotlin-lang.org)
 
+*NOTE: Development of this project is frozen*, as it will eventually be made obsolete by 
+[Arrow](arrow-kt.io), which merges the functionality of funKTionale and [Kategory](https://github.com/47deg/kategory-intro). If you are not yet using funKTionale, we recommend you [start with Arrow](http://arrow-kt.io/docs/) instead.
```

Comment:
The first URL is broken (missing scheme, so it turns into a relative path). I also suggest using the secure protocol for the third URL:

```suggestion
[Arrow](https://arrow-kt.io/), which merges the functionality of funKTionale and [Kategory](https://github.com/47deg/kategory-intro). If you are not yet using funKTionale, we recommend you [start with Arrow](https://arrow-kt.io/docs/) instead.
```

Reply:
Committed, thanks!